app/src/main/java/com/andela/art/settings/SettingsActivity.java

```diff
@@ -74,4 +62,16 @@ public boolean onOptionsItemSelected(MenuItem item) {
         super.onBackPressed();
         return true;
     }
+
+    @Override
+    public void onClick(DialogInterface dialogInterface, int whichOption) {
+        if (whichOption == AlertDialog.BUTTON_POSITIVE) {
+            FirebaseAuth.getInstance().signOut();
+            googleSignInClient.signOut();
+            Intent loginActivity = new Intent(SettingsActivity.this, LoginActivity.class);
+            startActivity(loginActivity);
```

Comment:
This can be shortened to;
`startActivity(new Intent(this, LoginActivity.class));`

Reply:
My new commit sets some flags on the `loginActivity` intent so I'll maintain this variable.